<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanelWidgetUI</class>
 <widget class="QWidget" name="ControlPanelWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ControlWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="5,4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,5">
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_Console">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_Status" stretch="2,5,1">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="vl_Toolbar">
         <item>
          <layout class="QHBoxLayout" name="hl_SerialConfig">
           <item>
            <widget class="QPushButton" name="pushButton_SerialConfig">
             <property name="text">
              <string>SerialConfig</string>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_SerialConnect">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelCOMstatus">
             <property name="text">
              <string>COM(N/A)@(N/A)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hl_CurrentMotor">
           <item>
            <widget class="QPushButton" name="pushButton_LoadMotorConfig">
             <property name="text">
              <string>LoadMotorConfig</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>CurrentMotor:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_CurrentMotor"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0">
         <item>
          <layout class="QGridLayout" name="gL_PID" rowstretch="0,0,0,0,0,0,0" columnstretch="1,2,1,0,1">
           <item row="5" column="2">
            <widget class="QLineEdit" name="lineEdit_Saved_Out_Limit"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Out_Limit</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLineEdit" name="lineEdit_Saved_I_Limit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>P</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Exact</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Saved</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>I_Limit</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>I</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="lineEdit_Saved_P"/>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="lineEdit_Saved_I"/>
           </item>
           <item row="3" column="2">
            <widget class="QLineEdit" name="lineEdit_Saved_D"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>D</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Precis.=1e1</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLabel" name="label_ExactP">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QLabel" name="label_ExactI">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="4">
            <widget class="QLabel" name="label_ExactD">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="4">
            <widget class="QLabel" name="label_ExactI_Limit">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="4">
            <widget class="QLabel" name="label_ExactOut_Limit">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hl_fetchPID">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_EnableFeedback">
           <property name="text">
            <string>EnableFeedBack</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_FetchPID">
           <property name="text">
            <string>FetchPID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_ApplyPID">
           <property name="text">
            <string>ApplyPID</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hL_Charts">
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
    </layout>
   </item>
  </layout>
  <action name="actionAdjustP">
   <property name="text">
    <string>AdjustP</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionadjustI">
   <property name="text">
    <string>adjustI</string>
   </property>
  </action>
  <action name="actionadjustD">
   <property name="text">
    <string>adjustD</string>
   </property>
  </action>
  <action name="actionadjustI_Limit">
   <property name="text">
    <string>adjustI_Limit</string>
   </property>
  </action>
  <action name="actionadjustOut_Limit">
   <property name="text">
    <string>adjustOut_Limit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
